Makefile

```diff
@@ -81,11 +81,11 @@ docker-build:
 docker-push:
 	docker push ${IMG}
 
-# Travis Latest Tag Deployment
-travis-latest-deploy: docker-login docker-build docker-push
+# CI Latest Tag Deployment
+ci-latest-deploy: docker-login docker-build docker-push
```

Comment:
@sabre1041 Should this PR go ahead and remove the `.travis.yml` file as well since these `make` targets will no longer be available to it?

Reply:
@kurtismullins I updated the PR to include travis and correct a few things. I havent had a chance to test the tag path fully, but since we arent planning a tag in the immediate, I'd like to get this out there